_rules/SC2-2-1+SC3-2-5-meta-refresh.md

```diff
@@ -110,6 +110,16 @@ First `<meta http-equiv="refresh">` element is not valid, second one redirects a
 </head>
 ```
 
+#### Passed example 4
+
+Redirects after exactly 20 hours.
+
+```html
+<head>
+  <meta data-rule-target http-equiv="refresh" content="72000; http://example.com" />
+</head>
+```
+
```

Comment:
This is not a pass, but a failure, as the time is neither 0 nor greater than 72000.

Reply:
Haha, it was definitely supposed to be a failure. It was in the right section (Failed test cases), but I forgot to change the heading after copy/pasting it over.
Should be fixed now